packages/node/src/evm/handlers/process-transactions.test.ts

```diff
@@ -38,14 +38,17 @@ describe('processTransactions', () => {
       hash: '0xad33fe94de7294c6ab461325828276185dff6fed92c54b15ac039c6160d2bac3',
     });
 
-    const apiCall = fixtures.requests.buildSubmittableApiCall({ requesterIndex: '4' });
-    const withdrawal = fixtures.requests.buildWithdrawal({ requesterIndex: '5' });
+    const apiCall = fixtures.requests.buildSubmittableApiCall({ sponsorAddress: '4' }); //TODO: fix value
+    const withdrawal = fixtures.requests.buildWithdrawal({ sponsorAddress: '5' }); //TODO: fix value
     const requests: GroupedRequests = {
       apiCalls: [apiCall],
       withdrawals: [withdrawal],
     };
-    const transactionCountsByRequesterIndex = { 4: 79, 5: 212 };
-    const state = fixtures.buildEVMProviderState({ requests, transactionCountsByRequesterIndex });
+    const transactionCountsBySponsorAddress = { 4: 79, 5: 212 }; //TODO: fix value
```

Comment:
Why the TODO here?

Reply:
I basically wasn't sure what sponsor address to use and where to get it so my main focus was to get the codebase to compile again first.

Reply:
Ah right. I forgot that this is by sponsor address now so the keys need to change